ftw/showroom/resources/showroom.js

```diff
@@ -432,7 +431,7 @@ module.exports = function Showroom() {
   function select(event) {
     event.preventDefault();
     var item = register.items.filter(function (item) {
-      return item.id === event.currentTarget.getAttribute("data-showroom-id");
+      return item.id === (event.currentTarget.getAttribute("data-showroom-id") || event.currentTarget.getAttribute("data-showroom-target-item"));
```

Comment:
if i understand correctly `data-showroom-id` and `data-showroom-target-item` are mutually exclusive, maybe we should enforce this somewhere?